client/my-sites/themes/theme-showcase.jsx

```diff
@@ -487,53 +486,70 @@ class ThemeShowcase extends Component {
 					title={ this.props.analyticsPageTitle }
 					properties={ { is_logged_in: isLoggedIn } }
 				/>
-				{ isLoggedIn && (
-					<ThemesHeader
-						description={ translate(
-							'Select or update the visual design for your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
-							{
-								components: {
-									learnMoreLink: <InlineSupportLink supportContext="themes" showIcon={ false } />,
-								},
-							}
-						) }
-					>
-						<div className="themes__install-theme-button-container">
-							<InstallThemeButton />
-						</div>
-						<ScreenOptionsTab wpAdminPath="themes.php" />
-					</ThemesHeader>
-				) }
+				<ThemesHeader
+					title={ isLoggedIn ? translate( 'Themes' ) : translate( 'WordPress themes' ) }
+					description={
+						isLoggedIn
+							? translate(
+									'Select or update the visual design for your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+									{
+										components: {
+											learnMoreLink: (
+												<InlineSupportLink supportContext="themes" showIcon={ false } />
+											),
+										},
+									}
+							  )
+							: translate(
+									'Beautiful and responsive themes for your photography site, portfolio, magazine, business website, or blog.'
+							  )
+					}
+				>
+					{ isLoggedIn && (
+						<>
+							<div className="themes__install-theme-button-container">
+								<InstallThemeButton />
+							</div>
+							<ScreenOptionsTab wpAdminPath="themes.php" />
+						</>
+					) }
+				</ThemesHeader>
 				<div className="themes__content" ref={ this.scrollRef }>
 					<QueryThemeFilters />
-					<SearchThemes
-						query={ filterString + search }
-						onSearch={ this.doSearch }
-						recordTracksEvent={ this.recordSearchThemesTracksEvent }
-					/>
-					{ tabFilters && (
-						<div className="theme__filters">
-							<ThemesToolbarGroup
-								items={ Object.values( tabFilters ) }
-								selectedKey={ this.state.tabFilter.key }
-								onSelect={ ( key ) =>
-									this.onFilterClick(
-										Object.values( tabFilters ).find( ( tabFilter ) => tabFilter.key === key )
-									)
-								}
+					<div className="themes__controls">
+						<div className="themes__controls-content">
```

Comment:
What's the usage of `<div className="themes__controls-content">`? I'm thinking it might not be necessary 🤔

Reply:
Yes, we can remove this. It's uncleaned code from the development of this PR 🙂